### ISSUE TITLE
Postbuch

### ISSUE THREAD

**Issue by j-dimension** [jlawyerorg/j-lawyer-org#79]
von einem Anwender:

"Abschließend hätte ich noch einen Vorschlag. Vorallem im Verwaltungsrecht ist ein Verzeichnis "Postbuch" des Postein- und Postausgangs sehr wichtig. Auch um das Nichtempfangen eines Schreibens beweisen zu können ist ein Postbuch unablässig. Registrieren sollte man u.a.:

    Name des Absenders
    Aktenzeichen
    Briefdatum
    Datum des Poststempels
    Empfangsdatum 

Das Postbuch hatte ich gestern vergessen zu erwähnen und scheint mir eine sehr wichtig Zufügung. Eine Anbindung des Postbuchs an das Wiedervorlage-Feature scheint mir sehr sinnvoll. "

**Comment by PBaumfalk** [comment]
Nicht zwingend, aber sehr hilfreich! https://www.iww.de/ak/kanzleiorganisation/kanzleiorganisation-postausgangsbuch-nicht-zwingend-aber-sinnvoll-f113513

**Comment by Thorsten-usr** [comment]
Hallo,
ja, auch ich unterstütze diese Ideen. Dabei habe ich weniger die offizielle Perspektive als eine subjektive. Ich möchte gegenüber den Mandanten stets belegen können, wann ich ihnen was habe zukommen lassen bzw. wann ich von ihnen welche Informationen erhalten habe, auch telefonisch. Momentan habe ich da ein Excel-Sheet mit einem Kommunikations-Journal.

Die Historie von J-Lawyer ist ok - allerdings mega redundant: Was da wann von wem in die Akte hinzugefügt wurde ist eine andere Information als die Kommunikation mit Außen. Und in der Vielzahl der automatischen Einträge verliere ich schnell die Übersicht. Insoweit ersetzt die Akten-Historie nicht das Postbuch oder ein Kommunikations-Journal.
Möglicherweise ist es sinnvoll, diese Informationen zu differenzieren.
Danke!

**Comment by Thorsten-usr** [comment]
Ja, gute Idee, das mit dem Versandtstatus von Dokumenten. Prinzipiell kommt man damit in den Bereich der "Versionierung" von Dokumenten. 

Mir persönlich wäre das aber zu komplex. Ich will die Dinge so einfach halten, wie möglich.
Ich habe mich jetzt von den Excel-Sheets als Kommunikations-Journal verabschiedet und nutze nur noch die von J-Lawyer angebotene Notizfunktion. Da mache ich eine "Kom-Journal"-Notiz und trage da händisch zeilenweise die wichtigsten Kommunikationsvorgänge ein. So eine Notiz läßt sich ad-hoc bearbeiten - ohne externes Programm - und enthält auch alle nötigen Angaben. Das reicht zumindest mir, um mich gegenüber Dritten zu vergewissern, daß und was ich verschickt habe. Einträge sollte man natürlich nur machen, wenn man auch wirklich zum Briefkasten o.ä. gegangen ist.

Eine Automatisierung dieses händischen Vorgehens ist aber natürlich immer zu begrüßen.
